client/components/forms/form-select/index.jsx

```diff
@@ -12,13 +12,15 @@ const FormSelect = React.createClass( {
 	},
 
 	render() {
-		const { className, isError, ...props } = this.props,
-			classes = classNames( className, 'form-select', {
-				'is-error': isError,
-			} );
+		const { inputRef, className, isError, ...props } = this.props;
+		const classes = classNames( className, 'form-select', {
+			'is-error': isError,
+		} );
+
+		const ref = inputRef ? { ref: inputRef } : {};
 
 		return (
-			<select { ...props } className={ classes }>
+			<select { ...props } { ...ref } className={ classes }>
```

Comment:
Would there be any difference if we chose not to spread and instead assigned `ref` even if it is `undefined`?

```jsx
<select { ...props } ref={ inputRef } className={ classes }>
```